test/Dapr.Actors.Test/Runtime/ActorRuntimeTests.cs

```diff
@@ -46,6 +46,17 @@ public void TestExplicitActorType()
             Assert.Contains(RenamedActorTypeName, actorRuntime.RegisteredActorTypes, StringComparer.InvariantCulture);
         }
 
+        [Fact]
+        public void TestRegisterOtherTypeAtRuntime()
+        {
+            var actorType = typeof(ActorRuntimeTests);
+            var actorRuntime = new ActorRuntime();
+
+            actorRuntime.RegisterActor(actorType);
+
+            Assert.DoesNotContain(actorType.Name, actorRuntime.RegisteredActorTypes, StringComparer.InvariantCulture);
+        }
+
```

Comment:
Add another test verifying the functionality which is being added.

Reply:
+1 Please add a case where it's successful.